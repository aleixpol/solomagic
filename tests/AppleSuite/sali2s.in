\ref something0
\mt somethingsomething salino somethingsomething

\ref something1
\mt somethingsomething saline somethingsomething

