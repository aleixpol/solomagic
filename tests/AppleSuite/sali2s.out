\ref something0
\mt somethingsomething salino somethingsomething
\ma somethingsomething salingno somethingsomething

\ref something1
\mt somethingsomething saline somethingsomething
\ma somethingsomething saline somethingsomething
